update.sh

```diff
@@ -10,4 +15,12 @@ latest="$(
 )"
 
 set -x
-sed -ri 's/^(ENV PIWIK_VERSION) .*/\1 '"$latest"'/' Dockerfile
+
+for variant in apache fpm; do
+	cp Dockerfile.template "$variant/Dockerfile"
+	sed -ri -e '
+		s/%%VARIANT%%/'"$variant"'/;
+		s/%%VERSION%%/'"$latest"'/;
+		s/%%CMD%%/'"${cmd[$variant]}"'/;
+	' "$variant/Dockerfile"
+done
```

Comment:
Maybe you are using a windows system? do you see this red char from GitHub interface? if you use unix system, and there is noline at the end, this shouldn't appear.